src/client/api/restful/get_message_children.c

```diff
@@ -185,3 +180,28 @@ int get_message_children(iota_client_conf_t const *ctx, char const msg_id[], res
   byte_buf_free(http_res);
   return ret;
 }
+
+void print_message_children(res_msg_children_t *res, uint8_t indentation) {
+  if (res == NULL) {
+    printf("[%s:%d] invalid parameters\n", __func__, __LINE__);
+    return;
+  }
+  if (res->is_error) {
+    printf("Error: %s\n", res->u.error->msg);
+  } else {
+    msg_children_t *data = res->u.data;
+    printf("%s{\n", PRINT_INDENTATION(indentation));
+    printf("%s\tmessageId: %s\n", PRINT_INDENTATION(indentation), data->msg_id);
+    printf("%s\tmaxResults: %d\n", PRINT_INDENTATION(indentation), data->max_results);
+    printf("%s\tcount: %d\n", PRINT_INDENTATION(indentation), data->count);
+    printf("%s\tchildrenMessageIds: [\n", PRINT_INDENTATION(indentation));
+    int len = utarray_len(data->children);
+    for (int i = 0; i < len; i++) {
+      printf(i > 0 ? ",\n" : "");
+      printf("%s\t\t%s", PRINT_INDENTATION(indentation), *(char **)utarray_eltptr(data->children, i));
+    }
+    printf("\n");
```

Comment:
There will be an empty line in empty array if there are no children message IDs.
Maybe you could also add an unit test where there is no children message IDs inside a JSON result?

Reply:
Done, handles case without children.